app/src/main/kotlin/at/bitfire/davdroid/resource/LocalResource.kt

```diff
@@ -93,4 +93,9 @@ interface LocalResource<in TData: Any> {
      */
     fun resetDeleted()
 
+    /**
+     * String representation of this local resource without implementation details
+     */
+    fun toDumpString() = "id=$id, fileName=$fileName, eTag=$eTag, scheduleTag=$scheduleTag, flags=$flags"
```

Comment:
I wonder how useful this information actually is. When a user views the debug info, it would be good if they could see details of the resource, like contact name, email address etc.

Is there a reason why you didn't override `toString`? I can image that a verbose `toString()` would be a good idea and then we could just use `toString()` for the debug info.